tests/common/libvm.sh

```diff
@@ -418,10 +418,17 @@ vm_wait_content_after_cursor() {
 EOF
 }
 
-vm_assert_journal_has_content() {
+# Minor helper that makes sure to get quoting right
+vm_get_journal_after_cursor() {
   from_cursor=$1; shift
+  to_file=$1; shift
   # add an extra helping of quotes for hungry ssh
-  vm_cmd journalctl --after-cursor "'$from_cursor'" > tmp-journal.txt
+  vm_cmd journalctl --after-cursor "'$from_cursor'" > $to_file
```

Comment:
I'm confused, how does the extra wrapper help?

Reply:
It's just a little helper that knows to double quote the cursor argument when doing `vm_cmd`, rather than keeping it in mind everywhere (and sometimes forgetting it like we did here in `test-autoupdate-check.sh`).

Reply:
Ah sorry, I was reading the commits out of order.  I see a new caller was added.  Makes sense!